<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-MEKL5HB" ignored="0" name="BankingAppTest(failed)" tests="5" failures="5" timestamp="2022-06-30T21:35:13 IST" time="97.282" errors="0">
  <testcase name="loginDDT" time="7.026" classname="com.inetBanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:35)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="7.916" classname="com.inetBanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:35)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="7.637" classname="com.inetBanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:35)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginTest" time="6.303" classname="com.inetBanking.testCases.TC_LoginTest_001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginTest_001.loginTest(TC_LoginTest_001.java:39)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="addNewcustomer" time="20.672" classname="com.inetBanking.testCases.TC_AddCustomerTest_003">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_AddCustomerTest_003.addNewcustomer(TC_AddCustomerTest_003.java:59)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- addNewcustomer -->
</testsuite> <!-- BankingAppTest(failed) -->
